common/lexer/flex_lexer_adapter.h

```diff
@@ -133,6 +154,9 @@ class FlexLexerAdapter : private CodeStreamHolder, protected L, public Lexer {
 
   // Contains the enumeration and the substring slice of the last lexed token.
   TokenInfo last_token_;
+
+  // Used to track characters discarded from the end of the token text
+  size_t discarded_;
```

Comment:
You could also initialize to `= 0` here as well, instead of the constructor.

Reply:
Sure, I did just that.